### DISCUSSION TITLE
Determinig the command run on a host from host output

### DISCUSSION THREAD

**Discussion by erkan-arslan** [ParallelSSH/parallel-ssh#387] [Q&A] [answered]
Consider we run commands on different hosts with ParallelSSHClient and client.run_command like;

HostA -> Command01 --Arg01
HostA -> Command01 --Arg02
HostB -> Command01 --Arg01
HostB -> Command01 --Arg02

from output structure I am able to undestand which host's output I am dealing with. But, from same host, I am not able to track which command output it is. Since the output formats of the command01 is similar with Arg01 and Arg02 flags, I am not able to differentiate  it. But, in order to feed correct info to DB I must know the flag used. 

Best regards,

How can I determine/understand both host and command from host output?

**Comment by pkittenis** [comment, marked as answer]
This is now provided as `HostOutput.fully_qualified_command` as of [2.16.0](https://github.com/ParallelSSH/parallel-ssh/releases/tag/2.16.0) to make things easier.
